packages/syft/src/syft/core/tensor/autodp/gamma_tensor.py

```diff
@@ -582,6 +582,243 @@ def sum(
 
         return result
 
+    def trace(
```

Comment:
MAYBE: could we add some test suites for testing the trace,min,max operations at Pointer level ?